### PR TITLE
[Ryujinx.Graphics.GAL] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs [hunk 1]
```diff
@@ -160,5 +160,20 @@ bool IEquatable<TextureCreateInfo>.Equals(TextureCreateInfo other)
                    SwizzleB == other.SwizzleB &&
                    SwizzleA == other.SwizzleA;
         }
+
+        public override bool Equals(object obj)
+        {
+            return obj is TextureCreateInfo info && ((IEquatable<TextureCreateInfo>)this).Equals(info);
```
Comment (gdkchan) [inline src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs:166]: Why is this cast to `IEquatable<TextureCreateInfo>` needed? Doesn't it pick the right overload if you just call `Equals` directly?

File: src/Ryujinx.Graphics.GAL/Multithreading/CommandHelper.cs [hunk 2]
```diff
@@ -17,8 +17,8 @@ static class CommandHelper
     {
         private delegate void CommandDelegate(Span<byte> memory, ThreadedRenderer threaded, IRenderer renderer);
 
-        private static int _totalCommands = (int)Enum.GetValues<CommandType>().Max() + 1;
-        private static CommandDelegate[] _lookup = new CommandDelegate[_totalCommands];
+        private static readonly int _totalCommands = (int)Enum.GetValues<CommandType>().Max() + 1;
```
[the remark this answers is not in the document]
Comment (TSRBerry) [reply src/Ryujinx.Graphics.GAL/Multithreading/CommandHelper.cs:20]: I commented about the default in another PR, but I need to search for it again.
There is no default defined for them. Rider defaults to UpperCamelCase for them, but AcK said, he'd prefer _lowerCamelCase instead.

File: src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs (hunk 1 above)
Comment (TSRBerry) [reply src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs:166]: image
If I do that I get this complaint, but for some reason it's gone if I add this cast again.
image

I'd like to compare the IL for that, but I can't find the Equals method in there at all.

**Review by marysaka — APPROVED** [review]

File: src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs (hunk 1 above)
Comment (gdkchan) [reply src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs]: Maybe it works if you replace `bool IEquatable<TextureCreateInfo>.Equals(TextureCreateInfo other)` with `public bool Equals(TextureCreateInfo other)`? I'm not sure why the interface is explicitly implemented here. THere is similar code in a lot of other places, and none of them explicitly implements the interface.

Comment (TSRBerry) [reply src/Ryujinx.Graphics.GAL/TextureCreateInfo.cs]: Yup that works, the warning is now also gone without the cast!

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Pull request merged**